shared/tracker/index.js

```diff
@@ -154,3 +157,34 @@ export function selector (username: string): (store: Object) => Object {
     }
   }
 }
+
+export type RemoteActions = {
+  onClose: (username: string) => void,
+  startTimer: () => void,
+  stopTimer: () => void
+}
+
+export function remoteComponentActions (dispatch: Dispatch) : RemoteActions {
+  return bindActionCreators({
+    onClose,
+    startTimer,
+    stopTimer
+  })
+}
+
+export function remoteComponentProps (username: string, store: Object, remoteActions: RemoteActions): Object {
+  return {
```

Comment:
the actions are plumbed through to us, this way this thing doesn't need to know about the remote manager and it can be cross platform (never called from mobile)